SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability in Hydra, we ask that you report it to us confidentially. Please email us at security@iohk.io with the details of the vulnerability and steps to reproduce it. Please use "Hydra Security Disclosure" as the subject line for your email.
+
+We will acknowledge receipt of your email and will aim to provide a timeline for remediation. We ask that you do not publicly disclose the vulnerability until we have had sufficient time to address it.
+
+We encourage responsible disclosure practices, which means giving us a reasonable opportunity to fix the issue before making any information public. We also ask that you do not abuse the vulnerability or attempt to exfiltrate any sensitive data while we are working on remediation.
+
+## Supported Versions
+
+We will provide security updates for the latest version of Hydra.
+
+## Reporting Channels
+
+You can report a security vulnerability by emailing security@iohk.io with the details of the vulnerability and steps to reproduce it. Please use "Hydra Security Disclosure" as the subject line for your email.
```

Comment:
Seems like this sentence is redundant with the one in the first section?

Reply:
That is, indeed. I didn't see it as a problem but it might be painful then to maintain so I'll take a look how to improve it.